pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -251,19 +266,26 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	stillRunning := time.NewTicker(25 * time.Second)
 	ps.StillRunning(agentName, warningTime, errorTime)
 
-	model := hardware.GetHardwareModel(log)
+	model = hardware.GetHardwareModel(log)
 	log.Noticef("Got HardwareModel %s", model)
 
 	var displayFunc DisplayFunc
 	var initFunc InitFunc
 	var arg string
 	var isDisplay bool
+
 	setFuncs := func(m modelToFuncs) {
 		displayFunc = m.displayFunc
 		initFunc = m.initFunc
 		arg = m.arg
 		isDisplay = m.isDisplay
+		if m.model == "SIEMENS AG.SIMATIC IPC127E" {
```

Comment:
Since we already have the model array to control whether this gets used I don't think you need an explicit comparison on the model here.

Reply:
Ok will remove that explicit check.

Reply:
I saw that explicit model check in some other place as well.

Reply:
Yes that is required because we do not want to execute the call for setting blinks for al models, but at the same time for debugging purpose we will have debug log with info on app status for all other platforms